packages/teleport/src/console/components/DocumentSsh/useSshSession.ts

```diff
@@ -91,10 +91,12 @@ function handleTtyConnect(
   session: Session,
   docId: number
 ) {
-  const { hostname, login, sid, clusterId } = session;
+  const { hostname, login, sid, serverId, clusterId } = session;
   const url = cfg.getSshSessionRoute({ sid, clusterId });
   ctx.updateSshDocument(docId, {
-    title: `${login}@${hostname}`,
+    // DELETE IN 5.0.0
```

Comment:
Are we sure that this is an upgrade issue similarly to clusterId and not one of the valid use cases (when hostname is not-specified) ? If not, then lets remove this comment.

Reply:
reverted, handled by webapi server: gravitational/teleport#4027